books_to_scrape/spiders/books.py

```diff
@@ -0,0 +1,53 @@
+import re
+from pathlib import Path
+from types import new_class
+
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
The imports related to Selenium (`webdriver`, `By`, `EC`, `WebDriverWait`) are not used in the current implementation. If Selenium is not intended to be used, these imports should be removed.